src/scancode/cli.py

```diff
@@ -461,8 +468,14 @@ def scancode(ctx,
         if not quiet:
```

Comment:
I do not get where and when the post scan plugins are invoked.... can you help me there?